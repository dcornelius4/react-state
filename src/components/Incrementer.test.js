import React from 'react';
import { shallow } from 'enzyme';
import Incrementer from './Incrementer';

describe('Incrementer Component', () => {
  it('renders the', () => {
    const wrapper = shallow(<Incrementer />);
    expect(wrapper).toMatchSnapshot();
  });
});
