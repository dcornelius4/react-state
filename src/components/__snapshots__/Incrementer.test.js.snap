// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Incrementer Component renders the 1`] = `
<Fragment>
  <p>
    Current Count: 
    0
  </p>
  <button
    onClick={[Function]}
  >
    Get it!
  </button>
</Fragment>
`;
